packages/zoe/test/unitTests/contracts/test-zoeHelpersWZcf.js

```diff
@@ -0,0 +1,129 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@agoric/install-ses';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import test from 'ava';
+
+import { E } from '@agoric/eventual-send';
+import bundleSource from '@agoric/bundle-source';
+
+// noinspection ES6PreferShortImport
+import { makeZoe } from '../../../src/zoeService/zoe';
+import { setup } from '../setupBasicMints';
+import fakeVatAdmin from './fakeVatAdmin';
+import { swap } from '../../../src/contractSupport';
+import { assertPayoutAmount } from '../../zoeTestHelpers';
+
+const contractRoot = `${__dirname}/zcfTesterContract`;
+
+test(`zoeHelper with zcf - swap`, async t => {
+  const {
+    moolaIssuer,
+    moola,
+    moolaMint,
+    simoleanIssuer,
+    simoleanMint,
+    simoleans,
+  } = setup();
+  const zoe = makeZoe(fakeVatAdmin);
+
+  // pack the contract
+  const bundle = await bundleSource(contractRoot);
+  // install the contract
+  const installation = await zoe.install(bundle);
+
+  // Alice creates an instance
+  const issuerKeywordRecord = harden({
+    Pixels: moolaIssuer,
+    Money: simoleanIssuer,
+  });
+
+  // This contract gives ZCF as the contractFacet for testing purposes
+  const { creatorFacet, publicFacet } = await E(zoe).startInstance(
+    installation,
+    issuerKeywordRecord,
+  );
+  /** @type ContractFacet */
+  const zcf = creatorFacet;
+
+  const seat1 = await zoe.offer(
```

Comment:
Can we always treat `zoe` as if it is remote, even though there are not multiple vats?
```suggestion
  const seat1 = await E(zoe).offer(
```

Reply:
Yep, good idea.